TestApp/Search/Sources/SearchViews/SearchScreen.swift

```diff
@@ -0,0 +1,17 @@
+import SwiftUI
+import Navigation
+import Search
+
+struct SearchScreen: View {
+
+    let searchResults: () -> Void
+
+    var body: some View {
+        VStack {
+            Button("Go to results") {
+                searchResults()
+            }
+        }
+        .navigationTitle("Search")
+    }
+}
```

Comment:
SwiftUI previews could be included in some screens so that we can see how the environment can be mocked.